### PR TITLE
samples: ei_data_forwarder: Add EI data forwarder sample

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds sample demonstrating usage of Edge Impulse data forwarder.

Jira: NCSDK-8038

### CODE REVIEW

**Comment by MarekPieta** [comment]
We can disable sensor_sim trigger in Kconfig after this fix goes in:
https://github.com/nrfconnect/sdk-nrf/pull/3816

File: samples/ei_data_forwarder/src/main.c [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	SENSOR_CHAN_ACCEL_Y,
+	SENSOR_CHAN_ACCEL_Z
+};
+
+static const struct device *sensor_dev;
+static const struct device *uart_dev;
+static atomic_t uart_busy;
+
+
+void uart_cb(const struct device *dev, struct uart_event *evt, void *user_data)
```
Comment (pdunaj) [inline samples/ei_data_forwarder/src/main.c:29]: static

**Review by pdunaj — CHANGES_REQUESTED** [review]

File: samples/ei_data_forwarder/src/main.c (hunk 1 above)
Comment (MarekPieta) [reply samples/ei_data_forwarder/src/main.c:29]: done

**Review by pdunaj — APPROVED** [review]

**Comment by MarekPieta** [comment]
The commit will be added as part of: https://github.com/nrfconnect/sdk-nrf/pull/3768.
I close this PR.

**Pull request closed without merging**